### PR TITLE
[it_IT] Translate CONTRIBUTING: What makes a good pull request? (partial)

### PR DESCRIPTION
**Pull request by FabrizioPe**

### CODE REVIEW

File: docs/locale/it_IT/source/CONTRIBUTING.rst [hunk 1]
```diff
@@ -188,19 +188,19 @@ Se una PR ha superato tutte le verifiche ma non viene revisionata entro 72 ore (
...
-A continuación, intente :doc:`construyendo el proyecto <dev-setup/build>` en su entorno de desarrollo local para garantizar que todo esté configurado correctamente.
+Come prossimo passo, prova a :doc:`compilare il progetto <dev-setup/build>` nel tuo ambiente di sviluppo locale per assicurarti che tutto sia configurato correttamente.
 
-Caracteristicas de un buen PR
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Caratteristiche di una buona pull request
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  Un cambio a la vez. Ni cinco, ni tres, ni diez. Uno y solo uno. ¿Por qué? Porque limita el área de explosión del cambio. Si tenemos una regresión, es mucho más fácil identificar el compromiso culpable que si tenemos algún cambio compuesto que impacta más en el código.
+-  Un cambiamento alla volta. Non cinque, non tre, non dieci. Uno e uno solo. Perché? Perché limita l'area di impatto del cambiamento. Se si verifica un problema, è molto più facile identificare la commit responsabile rispetto a quando abbiamo cambiamenti complessi che riguardano una'area più estesa del codice.
```
Comment (FrancescoUrbano) [inline docs/locale/it_IT/source/CONTRIBUTING.rst]: Fix typo "una'area"

File: docs/locale/it_IT/source/CONTRIBUTING.rst [hunk 2]
```diff
@@ -188,19 +188,19 @@ Se una PR ha superato tutte le verifiche ma non viene revisionata entro 72 ore (
...
 
--  Incluya un enlace a la historia de JIRA para el cambio. ¿Por qué? Porque a) queremos rastrear nuestra velocidad para juzgar mejor lo que creemos que podemos entregar y cuándo y b) porque podemos justificar el cambio de manera más efectiva. En muchos casos, debería haber alguna discusión en torno a un cambio propuesto y queremos vincularlo desde el cambio en sí.
+-  Se c'è un issue corrispondente su GitHub, includi un link all'issue su GitHub nel riepilogo della PR e nel suo messaggio di commit. Perché? Perché ci saranno spesso discussioni aggiuntive relativamente a un cambiamento proposto o a un bug nell'issue. Inoltre, se usi una sintassi del tipo "Risolve #<numero dell'issue>" nel riepilogo e nel messaggio di commit della PR , l'issue su GitHub sarà automaticamente chiuso quando avrà luogo il merge della PR.
```
Comment (FrancescoUrbano) [inline docs/locale/it_IT/source/CONTRIBUTING.rst]: Fix comma: "della PR , l'issue" (no spaces before commas)

**Review by FrancescoUrbano — CHANGES_REQUESTED** [review]
Requested fix of a couple of minor typos

**Review by FabrizioPe — COMMENTED** [review]
fixed typos

**Review by FrancescoUrbano — APPROVED** [review]
LGTM

**Pull request merged**